README.md

```diff
@@ -73,6 +73,15 @@ Navigate to http://localhost:8081/maven2/ to see the content of our local maven
 
 When working on Mac you might need to increase the memory limit for docker even more as jenkins has 2 workers by default, which might build projects in parallel (you might need to try with 10~16 GB although this hasn't been measured exactly yet).
 
+To start monitoring (root privileges needed) execute
```

Comment:
It looks like root should not be needed here in general (unless you have configured docker in such a way that it always requires root but this is nothing special when compared to other `start-*` scripts)

Reply:
removed